src/mlx/warnings.py

```diff
@@ -39,12 +42,23 @@ def return_doxygen_warnings(self):
         print("{count} doxygen warnings found".format(count=self.doxygen_counter))
         return self.doxygen_counter
 
+    def check_junit_failures(self, line):
+        '''
+        Function for counting the number of JUnit warnings in a xmlfile.
+        The function returns the number of failing test cases found
+        '''
+        self.junit_counter += len(re.findall(junit_pattern, line))
```

Comment:
This only gives us a count of the warnings, correct? Do we want to log the names of the failed testcases as well?